### PR TITLE
Add Power VS cluster api support

### PR DESCRIPTION
**Pull request by Karthik-K-N**

### CODE REVIEW

File: manifests/image-references [hunk 1]
```diff
@@ -30,3 +30,7 @@ spec:
     from:
       kind: DockerImage
       name: registry.ci.openshift.org/openshift:gcp-cluster-api-controllers
+  - name: ibmcloud-cluster-api-controllers
+    from:
+      kind: DockerImage
+      name: registry.ci.openshift.org/openshift:ibmcloud-cluster-api-controllers
```
Comment (JoelSpeed) [inline manifests/image-references:36]: Is this in payload already?

**Review by JoelSpeed — COMMENTED** [review]
If the upstream powervs is ibmcloud, what's the upstream ibmcloud equivalent?

File: manifests/image-references (hunk 1 above)
Comment (Karthik-K-N) [reply manifests/image-references:36]: Its almost there, Once [this](https://github.com/openshift/ocp-build-data/pull/2072) PR is merged I think we would be seeing the image in payload

**Comment by Karthik-K-N** [comment]
> If the upstream powervs is ibmcloud, what's the upstream ibmcloud equivalent?

For both PowerVS and IBMCloud Platform the upstream cluster-api name ibmcloud, but intern we have [IBMVPCCluster](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/a65deedf0fe7c905c8600f36e5daeb7d81f8ae4d/templates/cluster-template.yaml#L28-L38) and [IBMPowerVSCluster](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/77d06720f48ef80d820eb038acc777c481d52fb1/templates/cluster-template-powervs.yaml#L25-L37).

**Comment by JoelSpeed** [comment]
/hold cancel
/lgtm
/assign @Fedosin 

Mike could you also PTAL and leave an approve if you're happy with this

**Comment by Karthik-K-N** [comment]
@JoelSpeed i have rebased the changes to remove merge conflicts and your lgtm label has been removed, Could you please take a look.

**Pull request merged**